src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -5701,7 +5701,8 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <WriteLinesToFile
         File="$(IntermediateOutputPath)$(CleanFile)"
         Lines="@(_CleanUniqueFileWrites)"
-        Overwrite="true" />
+        Overwrite="true"
+        WriteOnlyWhenDifferent="true" />
```

Comment:
This seems like a potentially noticeable change? I'm assuming you added this to make it not error when the files are the same.

Reply:
Correct.  /t:Clean then /t:Clean /question should pass.  In most cases, it was writing an empty file over itself.

Reply:
Alternative, I could condition on Delete, if delete task removed no files, then can skip WriteLines.